backend/amt.pm

```diff
@@ -0,0 +1,263 @@
+# Copyright © 2018 Marek Marczykowski-Górecki
```

Comment:
Please make that SUSE LLC if you don't mind :)

Reply:
Done.